shared/web/WEB-INF/pages/header.jsp

```diff
@@ -6,81 +6,35 @@
   To change this template use File | Settings | File Templates.
 --%>
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
-<html>
-<head>
-    <link rel="stylesheet" href="${contextPath}/static/css/bootstrap/bootstrap.css">
-    <link rel="stylesheet" href="${contextPath}/static/css/bootstrap/bootstrap-theme.css">
-
-    <style>
-        form {
-            text-align: center
-        }
+<%@ taglib prefix='c' uri='http://java.sun.com/jsp/jstl/core' %>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<c:set var="contextPath" value="${pageContext.request.contextPath}"/>
+<!DOCTYPE html>
+<html lang="en">
 
-        #header {
-            background-color: black;
-            color: white;
-            text-align: center;
-            padding: 5px;
-            height: 100px;
-        }
+<head>
+    <title>WeStory</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
 
-        #section {
-            padding: 1%;
-        }
+    <script src="${contextPath}/static/js/jquery-1.11.3.js"></script>
+    <script src="${contextPath}/static/js/bootstrap/bootstrap.js"></script>
+    <link rel="stylesheet" href="${contextPath}/static/css/bootstrap.css">
+    <link rel="stylesheet" href="${contextPath}/static/css/bootstrap-theme.min.css">
+    <link rel="stylesheet" href="${contextPath}/static/css/styles.css">
+</head>
 
-        #footer {
-            background-color: black;
-            color: white;
-            text-align: right;
-            padding: 5px;
-            width:100%;
-            height:80px;
-            position:absolute;
-            bottom:0;
-            left:0;
-        }
 
-        .btn-modified {
-            display: inline-block;
-            padding: 3px 7px;
-            margin-bottom: 0;
-            font-size: 14px;
-            font-weight: normal;
-            line-height: 1.42857143;
-            text-align: center;
-            white-space: nowrap;
-            vertical-align: middle;
-            -ms-touch-action: manipulation;
-            touch-action: manipulation;
-            cursor: pointer;
-            -webkit-user-select: none;
-            -moz-user-select: none;
-            -ms-user-select: none;
-            user-select: none;
-            background-image: none;
-            border: solid#337ab7;
-            border-radius: 4px;
-        }
+<body style="background-color: #265a88">
```

Comment:
The header.jsp is a modular file by its design since we include it other pages. Therefore dividing it into smaller pieces would be infeasible and harder to manage. Same applies for most of the bottom comments. Thanks for the suggestion tough!